models/public/retinaface-resnet50-pytorch/model.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from torch import load, save
+
+from models.retinaface import RetinaFace
+from data.config import cfg_re50
+
+def remove_module_prefix(original_weights, converted_weights):
+    checkpoint = load(original_weights, map_location='cpu')
+    ckpt = {k.replace('module.', ''): v for k, v in checkpoint.items()}
+    save(ckpt, converted_weights)
+
+
+def create_model(original_weights, converted_weights):
+    remove_module_prefix(original_weights, converted_weights)
```

Comment:
I mean slightly different.
You do not need rewrite state dict on the disk (just combain create model and load state dict in create model function with modification on the fly)

Reply:
Done